### PR TITLE
Added rules for json to editorconfig

### PR DESCRIPTION
**Pull request by Bravo555**

### CODE REVIEW

**Comment by Bravo555** [comment]
dafuq travis

File: .editorconfig [hunk 1]
```diff
@@ -8,3 +8,7 @@ charset = utf-8
 [*.{cpp, hpp, h}]
 indent_style = tab
 indent_size = 4
+
+[*.json]
+indent_style = tab
+indent_size = 2
```
Comment (kaizi99) [inline .editorconfig]: Does indent size even matter when using tabs?

Comment (Bravo555) [inline .editorconfig]: Yeah, you can define tab width.

Comment (kaizi99) [inline .editorconfig]: Then why 2 instead of 4?

Comment (Bravo555) [inline .editorconfig]: Repertoi-e made 2 spaces indents in previous PR. Also JSON files have usually more indentation levels. But we can have 4, makes no difference to me.

Comment (kaizi99) [inline .editorconfig]: Well then okay

**Comment by kaizi99** [comment]
How can I approve this lol

**Comment by Repertoi-e** [comment]
It's closed

**Pull request closed without merging**